lib/testScenario/postmanCollectionRunnerClient.ts

```diff
@@ -341,13 +343,11 @@ export class PostmanCollectionRunnerClient implements TestScenarioRunnerClient {
     this.auth(stepEnv.env);
     const item = new Item();
     item.name = step.step;
-    const path =
-      "/subscriptions/:subscriptionId/resourcegroups/:resourceGroupName/providers/Microsoft.Resources/deployments/{{deploymentName}}?api-version=2020-06-01";
```

Comment:
where does `deploymentName` come from before?

Reply:
From env. but env is not define. So the value is undefine. It's a bug before.